src/dotless.Test/Specs/RulesetsFixture.cs

```diff
@@ -135,5 +135,135 @@ public void IESlash9HackAllowedForNonEvaluatedValues()
             var expected = input;
             AssertLess(input, expected);
         }
+
```

Comment:
Checked the less.js handling of !important and it is currently broken and there is a bug logged for it.
http://less2css.org/#%7B%22less%22%3A%22.padding()%20%7B%20%5Cn%20%20padding%2B_%3A%2010px%20!important%3B%20%5Cn%7D%5Cn%5Cndiv%20%7B%5Cn%20%20.padding()%3B%5Cn%20%20.padding()%3B%5Cn%7D%22%7D

https://github.com/less/less.js/issues/2367

But checking with w3c validator there should be a single !important in the end so I will go with that.

Also, I found that I made a mistake, + and +_ should not be separate groups according to hos the less.js version does it so I will change this.